src/clusterfuzz/_internal/bot/tasks/utasks/__init__.py

```diff
@@ -240,6 +241,7 @@ def _start_web_server_if_needed(job_type):
     logs.error('Failed to start web server, skipping.')
 
 
+@logs.task_stage_context(logs.Stage.PREPROCESS)
 def tworker_preprocess_no_io(utask_module, task_argument, job_type,
```

Comment:
I'm confused. Why do we need to wrap multiple functions. For example tworker_preprocess_no_io calls _preprocess, why do both need to be wrapped?

Reply:
It doesn't. removing